src/pages/workspace/reimburse/WorkspaceReimburseView.js

```diff
@@ -64,140 +64,130 @@ const defaultProps = {
     reimbursementAccount: ReimbursementAccountProps.reimbursementAccountDefaultProps,
 };
 
-class WorkspaceReimburseView extends React.Component {
-    constructor(props) {
-        super(props);
-        this.state = {
-            currentRatePerUnit: this.getCurrentRatePerUnitLabel(),
-        };
-    }
-
-    componentDidMount() {
-        this.fetchData();
-    }
-
-    componentDidUpdate(prevProps) {
-        if (prevProps.policy.customUnits !== this.props.policy.customUnits || prevProps.preferredLocale !== this.props.preferredLocale) {
-            this.setState({currentRatePerUnit: this.getCurrentRatePerUnitLabel()});
-        }
+function WorkspaceReimburseView(props) {
+    const [currentRatePerUnit, setCurrentRatePerUnit] = useState('');
```

Comment:
Looks like previously, there was initial state for `currentRatePerUnit` but now it's empty?

Reply:
There could have been 2 ways in which this could have been dealt:
1. Putting it in the initial state and moving the initialisation down the function declaration to avoid eslint error.
2. setting the state in useEffect, as it will work based on dependency array and while mounting.

Since we already need to useEffect to set this value, hence the state will be set during mounting so we don't need to initialise separately, allowing us better readability and code organisation.

Reply:
Thanks for explanation! 👍